paasta_tools/contrib/check_manual_oapi_changes.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -uo pipefail
+
+diff_names=$(git diff origin/master --name-only)
+touched_schemas=$(
+    echo "$diff_names" |
+    grep 'paasta_tools/api/api_docs/oapi.yaml' |
+    wc -l)
+
+schema_touched=$(echo "$diff_names" | grep 'paasta_tools/api/api_docs/oapi.yaml')
+api_touched=$(echo "$diff_names" | grep 'paasta_tools/paastapi')
+
+if [ ! -z "$(echo "$diff_names" | grep paasta_tools/paastaapi)" -a -z "$schema_touched" ]; then
```

Comment:
`"$(echo "$diff_names" | grep paasta_tools/paastaapi)"` is the same as `"$api_touched"`